deploy/application/files/values.yaml.tpl

```diff
@@ -21,18 +21,27 @@ env:
   ${key}: ${value}
 %{ endfor ~}
 
+{%- if app_url_type == "path" %}
+targetGroupBinding:
+  enabled: ${enable_target_group_binding}
+  targetGroupARN: ${target_group_arn}
+{%- endif %}
+
+{%- if app_url_type == "subdomain" %}
 ingress:
   className: alb
   annotations:
-    alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS":443}]'
-    alb.ingress.kubernetes.io/scheme: internet-facing
-    alb.ingress.kubernetes.io/ssl-policy: ELBSecurityPolicy-TLS13-1-2-2021-06
-    alb.ingress.kubernetes.io/target-type: ip
-    external-dns.alpha.kubernetes.io/hostname: ${hostname}
+  alb.ingress.kubernetes.io/scheme: internet-facing
+  alb.ingress.kubernetes.io/target-type: ip
+  alb.ingress.kubernetes.io/group.name: ${service_name}
+  alb.ingress.kubernetes.io/listen-ports: '[{"HTTP":80,"HTTPS":443}]'
+  alb.ingress.kubernetes.io/ssl-redirect: '443'
+  alb.ingress.kubernetes.io/healthcheck-path: @{{ app_healthcheck_endpoint }}
```

Comment:
@DCamma Thanks, this is a nice to have!

You reminded me of [Pod Readiness Gates](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.8/deploy/pod_readiness_gate/). They have direct support by the load balancer controller.

Thinking how can this be supported by our platform (unrelated to the PR, but wanted to mention it)